content/guides/integrations/dropwizard.md

```diff
@@ -0,0 +1,135 @@
+
+{{% notice tip %}}
+For assistance setting up Prometheus, [Click here](/codelabs/prometheus) for a guided codelab.
+
+You can swap out any other exporter from the [list of Java exporters](/guides/exporters/supported-exporters/java)
+{{% /notice %}}
+
+## Adding dependencies
+If you’ve got a Maven application, you’ll need to add opencensus-contrib-dropwizard as a dependency.
+
+```xml
+<dependencies>
+    <dependency>
+        <groupId>io.opencensus</groupId>
+        <artifactId>opencensus-contrib-dropwizard</artifactId>                  
+        <version>0.17.0</version>
+    </dependency>
+</dependencies>
+```
+
+And you can find its latest version [\[here\]](https://mvnrepository.com/artifact/io.opencensus/opencensus-contrib-dropwizard).
+
+For Gradle add to your dependencies:
+```xml
+compile ‘io.opencensus:opencensus-dropwizard:0.17.0’
+```
+
+## Enable an exporter
+Add the following code snippet to your `<dependencies>...</dependencies>` node in `pom.xml`:
+
+```xml
+<dependency>
+    <groupId>io.opencensus</groupId>
+    <artifactId>opencensus-exporter-stats-prometheus</artifactId>
+    <version>0.17.0</version>
+</dependency>
+
+<dependency>
+    <groupId>io.prometheus</groupId>
+    <artifactId>simpleclient_httpserver</artifactId>
+    <version>0.4.0</version>
+</dependency>
+```
+We also need to expose the Prometheus endpoint say on address “localhost:8888” in order for Prometheus to scrape our application. Please add the following to our Java code.
+
+{{<highlight java>}}
+private static void setupOpenCensusAndPrometheusExporter() throws IOException {
+  // Register the Prometheus exporter
+  PrometheusStatsCollector.createAndRegister();
+
+  // Run the server as a daemon on address "localhost:8888"
+  HTTPServer server = new HTTPServer("localhost", 8888, true);
+}
+{{</highlight>}}
+
+## End to end example
+
+{{<highlight java>}}
+import java.util.Collections;
+import java.io.IOException;
+import io.opencensus.exporter.stats.prometheus.PrometheusStatsCollector;
+import io.prometheus.client.exporter.HTTPServer;
+
+public class YourClass {
+  // Create registry for Dropwizard metrics.
+  static final com.codahale.metrics.MetricRegistry 
+    codahaleRegistry = new com.codahale.metrics.MetricRegistry();
+
+  // Create a Dropwizard counter.
+  static final com.codahale.metrics.Counter 
+     requests = codahaleRegistry.counter("requests");
+
+  public static void main(String[] args) {
+
+    // Increment the requests.
+    requests.inc();
+
+    // Hook the Dropwizard registry into the OpenCensus registry
+    // via the DropWizardMetrics metric producer.
+    io.opencensus.metrics.Metrics.getExportComponent().getMetricProducerManager().add(
+          new io.opencensus.contrib.dropwizard.DropWizardMetrics(
+            Collections.singletonList(codahaleRegistry)));
+    
+    setupOpenCensusAndPrometheusExporter();
+    ...
```

Comment:
I forgot to mention that perhaps make this a comment since "..." won't compile
so
```java
//...Continue with the rest of your application here
```